spec/services/notifications/moderation_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe Notifications::Moderation, type: :service do
+  # last_reacted_at is within the bounds, and last_moderation_notification is within the bounds
+  let(:user1) do
+    create(:user, :trusted,
+           last_moderation_notification: 4.days.ago,
+           last_reacted_at: 1.day.ago)
+  end
+
+  # last_reacted_at is not within the bounds, and last_moderation_notification is not within the bounds
+  let(:user2) do
+    create(:user, :trusted,
+           last_moderation_notification: 2.days.ago,
+           last_reacted_at: 8.days.ago)
+  end
+
+  # last_reacted_at is within the bounds, and last_moderation_notification is not within the bounds
+  let(:user3) do
+    create(:user, :trusted,
+           last_moderation_notification: 2.days.ago,
+           last_reacted_at: 2.days.ago)
+  end
+
+  # user is not trusted
+  let(:user4) { create(:user, last_reacted_at: 2.days.ago) }
```

Comment:
You created this `let` but I don't see it used anywhere in this spec?

Reply:
@maestromac thanks for picking this up, I meant to write a test to chcek for the use case where a user that is not trusted does not get added to the list of available moderators. I've updated the spec for this.